doc/sphinx/addendum/extraction.rst

```diff
@@ -303,11 +303,18 @@ what ML term corresponds to a given axiom.
       by a :cmd:`Extraction Inline`. Hence a :cmd:`Reset Extraction Inline`
       will have an effect on the realized and inlined axiom.
 
-.. caution:: It is the responsibility of the user to ensure that the ML
-   terms given to realize the axioms do have the expected types. In
-   fact, the strings containing realizing code are just copied to the
-   extracted files. The extraction recognizes whether the realized axiom
-   should become a ML type constant or a ML object declaration. For example:
+   .. caution:: It is the responsibility of the user to ensure that the ML
+      terms given to realize the axioms do have the expected types. In
+      fact, the strings containing realizing code are just copied to the
+      extracted files. The extraction recognizes whether the realized axiom
+      should become a ML type constant or a ML object declaration. For example:
+
+   .. exn:: The term @qualid is already defined as foreign custom constant.
+
+      The :n:`@qualid` was previously used in a
+      :cmd:`Extract Foreign Constant` command. Using :cmd:`Extract Inlined Constant`
+      for :n:`@qualid` would override this command.
+
```

Comment:
This breaks the documentation. Note that you can check the produced documentation by clicking on the link next to the job `doc:refman: refman artifact` in the list of checks at the bottom of the pull request.

First, the "caution" block is meant to apply to both commands `Extract Constant` and `Extract Inlined Constant`, so there is no reason to indent it here. Another possibility is to move the block under `Extract Constant`, since it is the primary command.

Second, you have put the "error" block right in the middle of the "caution" block (notice the "For example" part); you should move it earlier, e.g., before the "note" block.

Third (it is not your fault, but you might just as well fix it along the way), the sentence "The extraction recognizes..." should be moved outside the "caution" block, next to the Coq script below. It could also be made into a "note" block (including the Coq script) and moved under `Extract Constant`.

Reply:
Thank you @silene. I will fix that. Should I also squeeze the changes with my previous commit and/or rebase on the current state of coq/master?

Reply:
Okay, now the layout looks good and my branch is in sync with coq/master. I'll wait for the checks to complete. Is the full CI done automatically?

Reply:
Made some small changes on the FFI section. I had some plain @string and @qualid strings. They should also be references. I'll check the refman when it is finished. I cannot check offline since the matching sphinx version is not supported on my system.